slacker.go

```diff
@@ -130,36 +112,6 @@ func (s *Slacker) Interactive(interactiveEventHandler func(InteractiveBotContext
 	s.interactiveEventHandler = interactiveEventHandler
 }
 
-// CustomBotContext creates a new bot context
-func (s *Slacker) CustomBotContext(botContextConstructor func(context.Context, *slack.Client, *socketmode.Client, *MessageEvent) BotContext) {
-	s.botContextConstructor = botContextConstructor
-}
-
-// CustomInteractiveBotContext creates a new interactive bot context
-func (s *Slacker) CustomInteractiveBotContext(interactiveBotContextConstructor func(context.Context, *slack.Client, *socketmode.Client, *socketmode.Event) InteractiveBotContext) {
-	s.interactiveBotContextConstructor = interactiveBotContextConstructor
-}
```

Comment:
Not clear to me why this was needed to begin with and recently introduced so im ok removing.